lib/eval_on_dart_library.dart

```diff
@@ -63,4 +96,103 @@ class EvalOnDartLibrary {
         print('Unrecognized error: $e');
     }
   }
+
+  Future<Library> getLibrary(LibraryRef instance, ObjectGroup isAlive) {
+    return getObjHelper(instance, isAlive);
+  }
+
+  Future<Class> getClass(ClassRef instance, ObjectGroup isAlive) {
+    return getObjHelper(instance, isAlive);
+  }
+
+  Future<Func> getFunc(FuncRef instance, ObjectGroup isAlive) {
+    return getObjHelper(instance, isAlive);
+  }
+
+  Future<Instance> getInstance(
+    FutureOr<InstanceRef> instanceRefFuture,
+    ObjectGroup isAlive,
+  ) async {
+    return await getObjHelper(await instanceRefFuture, isAlive);
+  }
+
+  /// Public so that other related classes such as InspectorService can ensure their
+  /// requests are in a consistent order with requests which eliminates otherwise
+  /// surprising timing bugs such as if a request to dispose an
+  /// InspectorService.ObjectGroup was issued after a request to read properties
+  /// from an object in a group but the request to dispose the object group
+  /// occurred first.
```

Comment:
Nit: Not sure if this was just from the port, but I'd suggest breaking this long sentence into multiple sentences and adding some commas so that it is easier to understand. i.e.

/// Public so that other related classes such as InspectorService can ensure their
/// requests are in a consistent order with existing requests. This eliminates otherwise
/// surprising timing bugs, such as if a request to dispose an
/// InspectorService.ObjectGroup was issued after a request to read properties
/// from an object in a group, but the request to dispose the object group
/// occurred first.

Reply:
All the comments are directly ported. Used your version of the comment instead.